scripts/ci/known-flakes-wrapper.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This is a wrapper script to re-run tests for known flakes.
+# Known flake error messages can be added to the array KNOWN_FLAKES
+# and the script will re-run the tests if one of the known flake
+# error messages appears.
+# The script will try to re-run the failing tests for $max_retries.
+# Most used CI systems have a time limit, so max_retries should
+# probably not be larger than 3. If a test fails for 3 times maybe
+# something really needs to be fixed.
+# Motivation for this script was that we currently just re-run CI
+# if we see a known flake. This script tries to automate the step
+# of automatically re-running CI for known flakes.
+
+if [ "$#" -ne 1 ]; then
+	echo "Usage: $0 <actual-script>" >&2
+	exit 1
+fi
+
+KNOWN_FLAKES[0]="page-xfer: Can't read pagemap from socket: Input/output error"
```

Comment:
Maybe we can have an issue link comments here? So that when problem is solved it would be easier to understand that flake is not needed anymore.

Reply:
Yes, having issues linked here would make totally sense.

Currently I see more TLS errors again. So I still would like something automatic instead of me hitting the 're-run' button.

Reply:
I still vote to fix issues rather than ignore them ;).